low-level/framework/sspl_ll_d

```diff
@@ -414,11 +430,15 @@ def _sensors_actuators_factory(sspl_threaded_modules, msgQlist, operating_system
     """Loops thru list of sensors/actuators and instantiate"""
 
     # Read in list of sensors
-    sensors = SSPL_SETTINGS.get("SENSORS")
+    sensors = []
+    for group in sspl_settings_configured_groups:
+        sensors.extend(SSPL_SETTINGS[group].get("SENSORS"))
```

Comment:
if no where printing final list, can we print here with INFO log level. Same for other module types being enabled finally